.eslintrc.js

```diff
@@ -22,27 +21,26 @@ module.exports = {
     // node files
     {
       files: [
-        'index.js',
-        'testem.js',
-        'ember-cli-build.js',
-        'config/**/*.js',
-        'tests/dummy/config/**/*.js',
-        'tests-node/**/*.js',
-        'lib/**/*.js',
+        './.eslintrc.js',
+        './.prettierrc.js',
+        './ember-cli-build.js',
+        './index.js',
+        './testem.js',
+        './blueprints/*/index.js',
+        './config/**/*.js',
+        './lib/**/*.js',
+        './tests/dummy/config/**/*.js',
+        './tests-node/**/*.js',
```

Comment:
The list here should match that from the latest blueprint (v4.4.0). The files that this addon lists, in addition, are:

- `./lib/**/*.js`
- `./tests-node/**/*.js`